prime-router/metadata/schemas/AZ/pima-az-covid-19-csv.schema

```diff
@@ -2,22 +2,23 @@
 name: pima-az-covid-19
 description: Pima County, AZ COVID-19 flat file
 topic: covid-19
-trackingElement: standard.testing_lab_specimen_id
+trackingElement: testing_lab_specimen_id
+basedOn: covid-19
```

Comment:
The Pima Schema is actually 99% the same as the SimpleReport schema.  (I created it by first just copying the pdi schema, then making adjustments.)

So, again, this is subtly different, but maybe we should be doing something like this:
```
  name: pima-az-covid-19
  description: Pima County, AZ COVID-19 flat file
  data-dictionary: covid-19
  extends:   pdi-covid-19
```
That way the pima schema can make use of all the fancy Valuesets etc in the data dictionary, while guaranteeing its just a subset of data coming from SimpleReport.